sdk/cosmos/pom.xml

```diff
@@ -358,57 +378,31 @@
   </build>
     <reporting>
         <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-report-plugin</artifactId>
-                <version>2.22.0</version>
-            </plugin>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>findbugs-maven-plugin</artifactId>
-                <version>3.0.4</version>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-jxr-plugin</artifactId>
-                <version>2.1</version>
-            </plugin>
+            <!--<plugin>-->
```

Comment:
Commented out configurations?

Reply:
Yeah didn't remove them, if Cosmos team needs it, they can enable it back. But its not needed for compilation or running tests.

Reply:
Removed.